events/tests/test_event_get.py

```diff
@@ -192,6 +192,30 @@ def test_get_event_list_verify_bbox_filter(api_client, event, event2):
     )
 
 
+@pytest.mark.django_db
+def test_get_event_list_verify_dwithin_filter(event, event2):
+    origin = Point(24, 24)
+    # Sanity check for distances
+    assert origin.distance(event.location.position) > 36
+    assert origin.distance(event2.location.position) < 34
+
+    # Don't filter if missing either origin or metres.
+    get_list_and_assert_events(f"dwithin_origin={origin.x},{origin.y}", [event, event2])
+    get_list_and_assert_events("dwithin_metres=35", [event, event2])
+
+    get_list_and_assert_events(
+        f"dwithin_origin={origin.x},{origin.y}&dwithin_metres=35", [event2]
```

Comment:
Throwing in garbage non-numeric values produces 500 internal server error instead of  400 bad request..  although I don't know what is the general state of validation in this project. The above test on bbox does return 400 "bbox values must be floating points or integers", although that's thanks to munigeo doing the validation.

Reply:
Yeah, I spotted that as well, wasn't quite sure whether to add error handling or not. I suppose I could add ParseErrors or something 🤔

Reply:
500s will end up in Sentry, and we certainly don't want that 🙂